src/_cffi_src/openssl/ssl.py

```diff
@@ -688,6 +689,12 @@
 
 static const long Cryptography_HAS_SSL_CTX_CLEAR_OPTIONS = 1;
 
+#ifdef OPENSSL_NO_TLSEXT
```

Comment:
Rethinking that, I'm fine with removing it... But this means PyPy cannot support any openssl version that would set the macro?